contrib/crio-wipe/README.md

```diff
@@ -0,0 +1,27 @@
+# crio-wipe
+
+crio-wipe is a program that reads CRI-O's version file, and compares it to the output of crio --version.
+If the version is deemed old enough, crio-wipe wipes container storage on the node.
+If there is no version file present, or the version file is malformatted, crio-wipe also will wipe the storage.
+
+
+crio-wipe returns 0 if no wipe happend, 1 if a wipe did happen, or 2 if an error occured that resulted in no wipe.
```

Comment:
We should return 0 on success (whether we wiped or not). We could use logging to indicate whether a wipe happened or not.